recipes/mne-connectivity/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "mne-connectivity" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d88ceeb87953737e6c3ac5ba866447a5ddab0a276752fd9c69a00ea3db86e453
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+
+  run:
+    - python >=3.7
+    - mne-base >=0.24
+    - numpy
+    - scipy
+    - xarray
+    - netCDF4
+    - pandas
+    - scikit-learn
+    - pyvista >=0.30
+    - pyvistaqt >=0.4
+    - sip
+    - pyqt
+    - pyqt5-sip
```

Comment:
These requirements don't align with what upstream has. Are you sure they are correct?

Reply:
The specification of the upstream requirements is a bit sloppy in `setup.py`; the "actual" deps are in [`requirements.txt`](https://github.com/mne-tools/mne-connectivity/blob/main/requirements.txt):

```
numpy
scipy
mne>=0.24
xarray
netCDF4
h5netcdf
pooch
tqdm
matplotlib
pyqt5>=5.10,<5.14; platform_system == "Darwin"
pyqt5>=5.10,!=5.15.2,!=5.15.3; platform_system == "Linux"
pyqt5>=5.10,!=5.15.3; platform_system != "Linux" and platform_system != "Darwin"
pyqt5-sip
sip
pyvista>=0.30
pyvistaqt>=0.4
pandas
```

cc @larsoner could you confirm this?

Reply:
But I see I missed a few, I'll update

Reply:
@BastianZim I think with 829b34a now the dependencies of this package are in sync with upstream.

Reply:
You are right, but `scikit-learn` is listed in their installation instructions:
https://mne.tools/mne-connectivity/dev/install.html

It seems everything is slightly out of sync here :\ (setup.py, requirements.txt, installation instructions)

Reply:
Ahh ok thanks. There, minimum versions are listed, those should probably be included here.

Reply:
I have no idea if those listed are up to date. But since they usually don't **decrease** as time passes, I think I'll just list them here, alright